apps/prairielearn/src/question-servers/freeform.js

```diff
@@ -1305,17 +1284,36 @@ export async function renderAsync(
     locals.question_renderer = context.renderer;
 
     return withCodeCaller(context.course_dir_host, async (codeCaller) => {
-      await async.series([
-        // FIXME: support 'header'
-        async () => {
-          if (!renderSelection.question) return;
+      // FIXME: support 'header'
```

Comment:
@mwest1066 this comment blames back to you. As far as I can tell, this is a remnant of a WIP version of v3 questions where we had a separate `header` panel, which of course doesn't exist today. I think we can remove this, but can you confirm

Reply:
Removed in b0a7b5a30. Given that this has been around for years and never affected functionality I believe this is not useful.